commit 6d0ea02a4942a8454cf733669f3ecea9899532cb (refs/changes/85/2285/2)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-07-27T20:54:53+00:00 (2 years, 2 months ago)
    
    VOLAPI-62 Remove all references to state `rolling_back` in current implementation
    Reviewed by: Josh Wilsdon <jwilsdon@joyent.com>
    Approved by: Josh Wilsdon <jwilsdon@joyent.com>

diff --git a/lib/validation/volumes.js b/lib/validation/volumes.js
index a77c3c3..1e14d2b 100644
--- a/lib/validation/volumes.js
+++ b/lib/validation/volumes.js
@@ -148,9 +148,7 @@ function validateVolumeSizeSearchParam(size) {
 function validateVolumeState(state) {
     console.log('state: ', state);
 
-    var VALID_STATES = [
-        'creating', 'ready', 'failed', 'rolling_back', 'deleting'
-    ];
+    var VALID_STATES = ['creating', 'ready', 'failed', 'deleting'];
     var err;
 
     if (VALID_STATES.indexOf(state) === -1) {
